recipes/novas/LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
This looks like it should be public domain with the license file being:
https://github.com/brandon-rhodes/python-novas/blob/2e690e925e53ed585f57b3105cf92ea70954b60e/README-PyPI#L210-L226

Reply:
You are very correct, done.